analyses/NM_cd_2000/doc_NM_cd_2000.md

```diff
@@ -0,0 +1,28 @@
+# 2000 New Mexico Congressional Districts
+
+## Redistricting requirements
+In New Mexico, districts must:
+
+1. be contiguous
+1. have equal populations
+1. be geographically compact
+1. preserve county and municipality boundaries as much as possible
+
+The house of representatives is composed of seventy members to be elected from districts that are contiguous and that are as compact as is practical and possible.
```

Comment:
This should just be focused on the 2000 congressional districts, not the state legislative districts.